api/lib/infrastructure/repositories/jury-certification-repository.js

```diff
@@ -131,9 +137,7 @@ async function _toDomainWithComplementaryCertifications({
 
 function _toComplementaryCertificationCourseResultForJuryCertification(complementaryCertificationCourseResults) {
   const [complementaryCertificationCourseResultsWithExternal, commonComplementaryCertificationCourseResults] =
-    _.partition(complementaryCertificationCourseResults, (ccr) => {
-      return ccr.partnerKey.startsWith('PIX_EDU');
-    });
+    _.partition(complementaryCertificationCourseResults, ({ hasExternalJury }) => hasExternalJury);
```

Comment:
Pour faire plus court avec lodash:
```suggestion
    _.partition(complementaryCertificationCourseResults, 'hasExternalJury');
```

Reply:
Done.